src/app/modules/webhook/webhook.service.ts

```diff
@@ -223,9 +223,13 @@ export const sendWebhook = (
 }
 
 export const getWebhookType = (webhookUrl: string) => {
-  const isZapier = webhookUrl.match(/^https:\/\/hooks\.zapier\.com\//)
-  const isPlumber = webhookUrl.match(/^https:\/\/plumber\.gov\.sg\/webhooks\//)
-  const webhookType = isZapier ? 'zapier' : isPlumber ? 'plumber' : 'generic'
+  const isZapier = new RegExp(/^https:\/\/hooks\.zapier\.com\//)
+  const isPlumber = new RegExp(/^https:\/\/plumber\.gov\.sg\/webhooks\//)
```

Comment:
/nit: the `new RegExp()` constructor call is not needed. I recommend only using it if you construct regexp patterns from parts. Otherwise, stick to the litteral syntax:
```suggestion
  const isZapier = /^https:\/\/hooks\.zapier\.com\//
  const isPlumber = /^https:\/\/plumber\.gov\.sg\/webhooks\//
```